### PR TITLE
Added a lookup object instead of if statements

### PR DESCRIPTION
**Pull request by bengarvey**
Took some feedback from Elise on how to improve this function.

No tests ended up in the final PR, but I tested that the output of the old and new functions are the same in the raw dataset.

### CODE REVIEW

File: convert.js [hunk 1]
```diff
@@ -75,44 +75,27 @@ function getVote(rec) {
...
-  }
-  else if (meta[0] === 'Federal') {
-    return 'scope';
-  }
-  else if (meta[0] === 'Write-in') {
-    return 'columns2';
-  }
-  else if (meta[0] === 'Camden') {
-    return 'county';
+  const match = meta[0];
```
Comment (EliseWei) [inline convert.js]: Could include a check that meta has a length, for safety's sake.

File: convert.js [hunk 2]
```diff
@@ -75,44 +75,27 @@ function getVote(rec) {
...
-  }
-  else if (meta[0] === 'CAM_20181106_E') {
-    return 'code';
-  }
-  else {
-    return 'office';
-  }
-}
 
+  return typeof(lookup[match]) !== 'undefined' ? lookup[match] : 'office';
```
Comment (EliseWei) [inline convert.js:99]: Could be
`return lookup[match] || 'office';`
if we're not worried about the value of lookup[match] being falsey. Otherwise, 👍🏽

**Review by EliseWei — COMMENTED** [review]
Awesome update! Much more maintainable, I think.

**Comment by bengarvey** [comment]
@EliseWei thank you!

**Pull request merged**